src/mlpack/methods/adaboost/adaboost_model.hpp

```diff
@@ -8,7 +8,8 @@
 #define MLPACK_METHODS_ADABOOST_ADABOOST_MODEL_HPP
 
 #include <mlpack/core.hpp>
-#include "adaboost.hpp"
+
+class AdaBoost;
```

Comment:
Could you please add a comment here indicating that `adaboost.hpp` isn't included to accelerate compilation?